<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWindow</class>
 <widget class="QMainWindow" name="ChatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chat Window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="question_label">
      <property name="text">
       <string>Question:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="question_box"/>
    </item>
    <item>
     <widget class="QPushButton" name="ask_button">
      <property name="text">
       <string>Ask</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="reply_label">
      <property name="text">
       <string>Reply:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="reply_box">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
